edb/edgeql/compiler/func.py

```diff
@@ -214,6 +214,18 @@ def compile_FunctionCall(
         ctx=ctx,
     )
 
+    if func_name.module != 'std':
```

Comment:
Let's check whether the function has a definition in edgeql instead; (`nativecode`)